src/components/ProductCardList.jsx

```diff
@@ -1,17 +1,33 @@
 import ProductCard from './ProductCard';
+import { Skeleton } from '@nextui-org/react';
+
+const ProductCardList = ({
+  products,
+  title,
+  isHorizontalViewInMobile = true,
+  isLoading,
+  skeletonNo = 5
+}) => {
+  const className = isHorizontalViewInMobile ? 'flex overflow-x-auto' : 'grid';
```

Comment:
`isHorizontalViewInMobile` determines how the ProductCard is displayed on mobile devices. If set to true, ProductCards will be shown in a horizontal scroll view. If false, they will be arranged in a grid layout.